packages/fx-core/tests/component/driver/aad/create.test.ts

```diff
@@ -482,9 +482,9 @@ describe("aadAppCreate", async () => {
     expect(endTelemetry.properties.success).to.equal("no");
     expect(endTelemetry.properties["error-code"]).to.equal("aadAppCreate.HttpClientError");
     expect(endTelemetry.properties["error-type"]).to.equal("user");
-    expect(endTelemetry.properties["error-message"]).to.equal(
-      'A http client error happened while performing the aadApp/create task. The error response is: {"error":{"code":"Request_BadRequest","message":"Invalid value specified for property \'displayName\' of resource \'Application\'."}}'
-    );
+    // expect(endTelemetry.properties["error-message"]).to.equal(
```

Comment:
will there be another property to store error message?

Reply:
We will use another property name after we can make sure that the error message is in our total control. Retire this for now.